management/actions.go

```diff
@@ -200,6 +200,21 @@ type ActionExecution struct {
 	UpdatedAt *time.Time `json:"updated_at"`
 }
 
+// ActionLogSessionFilter defines a filter for the log session.
+type ActionLogSessionFilter struct {
+	Key *string `json:"key,omitempty"`
+	Val *string `json:"val,omitempty"`
+}
```

Comment:
Is there a package convention of _always_ using pointer values? I've seen this a lot in things like the AWS SDK, but I believe its to differentiate between the zero value (which could be a valid value) and _no_ value.

You would never _not_ specify a key _and_ value if you were providing a filter (to my understanding). So--assuming this isn't against package conventions--I would change `*string` to `string` and get rid of `omitempty`.

Reply:
From what I can tell from sampling the different definitions, pointers are _always_ used - akin to the AWS SDK as you mentioned, there are also helpers to wrap/unwrap values `auth0.String()`.

I agree with you, I find the pointer a bit strange for the `Filters` since those should always be defined. I'll leave this comment feedback open for now and maybe someone from the SDKs team can chime in and I'll apply the feedback if so 👌

Reply:
The reason we have pointers on most of the fields (but not every single one of them) is because we need to differentiate between e.g. an empty string, a field getting omitted from the payload and the field getting set to whatever string value, this is due to the way the Management API works on most endpoints. 

However in this specific case if the key and value are always set and we can never update just partially an `ActionLogSession`, it makes sense to not use the pointers. As for the `omitempty` tag, can these values be set to an empty string?

Also can we only create `ActionLogSession's` ? But not be able to read, update or delete?

Reply:
Hey @FadyMak 👋🏻 , any updates on this?:)

Reply:
Hey @sergiught 👋 Thanks for the clarification.

> Also can we only create ActionLogSession's ? But not be able to read, update or delete?

That's correct, an `ActionLogSession` can only be created - there are no other operations possible. The returned session URL from the endpoint's successful response can be used to tail the Actions logs.

> However in this specific case if the key and value are always set and we can never update just partially an ActionLogSession, it makes sense to not use the pointers.

Sounds good, I've converted the pointers to plain strings as both the `key` and `val` have to be defined ✅ 

> As for the `omitempty` tag, can these values be set to an empty string?

The only requirement from the Management API is that both the `key` and `val` properties are present when defining a filter. So, yes, they can be empty 👌 

I've also applied a change to remove the `omitempty`